charts/spire/charts/tornjak-frontend/templates/deployment.yaml

```diff
@@ -53,9 +53,9 @@ spec:
           {{- end }}
           volumeMounts:
             - name: cache
-              mountPath: /usr/src/app/node_modules/.cache
+              mountPath: {{ .Values.workingDir }}/node_modules/.cache
             - name: env
-              mountPath: /usr/src/app/build/tmp
+              mountPath: {{ .Values.workingDir }}/build/tmp
```

Comment:
Maybe default this to a path that works for Openshift, so we don't need that configurable.

Reply:
Yeah, I'd like to know a bit more why the change was needed. If the reasoning is compatible with hard coding it to the new location, that would be best. But it may imply other clusters may need it configured other ways, in which case it should be configurable. So, knowing why openshift needed this change is important.